pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -54,8 +54,8 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 		ticker         = time.NewTicker(5 * time.Second)
 	)
 	defer ticker.Stop()
-
-	timer := time.NewTimer(time.Duration(e.StageConfig.WaitApprovalStageOptions.Timeout) * time.Second)
+	timeout := time.Duration(e.StageConfig.WaitApprovalStageOptions.Timeout)
+	timer := time.NewTimer(timeout)
```

Comment:
just to confirm, the bug caused by this deleted `time.Second` annotation, is that right? 🤔 
(Timeout default value currently set as 6h already ref: [config/deployment](https://github.com/pipe-cd/pipe/blob/master/pkg/config/deployment.go#L25))